molgenis_documentation/technical_documentation.md

```diff
@@ -22,88 +22,36 @@ The “hairball” is the twisted, tangled maze of teaching that is going to tak
 * Contribution policies (if your project accepts outside contributions).
 
 
-# MOLGENIS Technical documentation
+# MOLGENIS user documentation
 ---
+## Introduction
```

Comment:
I agree, either demo server, war from maven, or the war we add to the release.
Compiling and running from your IDE should be in 'The Reference' section where people who are actually interested in developing should end up